Add Oh My Zsh, Romanian aspell dictionary, Montserrat font.

scolobb opened the pull request:
Here's my try at packaging three things I use every day:
- Oh My Zsh,
- Romanian spell dictionary,
- Montserrat font.

Is having these packages interesting to the Nix(OS) community?

This is my first time ever, so I am sorry for all slip-ups.
###### Things done:
- [?] Tested via `nix.useChroot`.   # Don't know how :-(
- [X] Built on platform(s): NixOS.
- [ N/A ] Tested compilation of all pkgs that depend on this change.   # Only new packages.
- [X] Tested execution of binary products.
- [Y] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).    # To me it looks like it does.

scolobb committed:
oh-my-zsh-git: init at 2016-02-27

scolobb committed:
aspell-dict-ro: init at 3.3-2

scolobb committed:
montserrat: init at 1.0

NeQuissimus commented:
Since you are running NixOS, you can simply define `nix.useChroot = true;` in your configuration.
You may also want to put yourself into the maintainers if you intend to keep these up-to-date.
I am certainly going to be using ohmyzsh, so you can put me into the maintainers along with yourself if you would like.

scolobb commented:
> Since you are running NixOS, you can simply define nix.useChroot = true; in your configuration.

Oh, sure, all right, thanks for pointing out :-) Done that, everything seems to build fine.

I added yourself and myself to the list of maintainers of `oh-my-zsh-git`, and also myself to the list of maintainers of `montserrat`.

NeQuissimus commented:
One more thing: Usually, the commit messages should be of the form `package: v1 -> v2` or `package: init at v1`, depending on whether this is an update or a new package.
Could you rebase for that?

Beyond that and assuming the packages work, everything looks good to me.

scolobb commented:
I fixed the commit messages.

Not sure I understand what Travis complains about.  To me it looks like the issue is unrelated to my modifications, but I'm not really sure.

scolobb commented:
Still not sure about the error from Travis; especially given the fact that the code has not changed from the previous commit, while the error message seems to have.

scolobb commented:
Switched to `sha256`, thank @vbgl.

edolstra commented on pkgs/shells/oh-my-zsh/default.nix:
```diff
@@ -0,0 +1,71 @@
+  echo "Oh My Zsh has been installed into $out/share/oh-my-zsh."
+  echo
+  echo "To use Oh My Zsh, run the following command:"
+  echo "  $ cp $outdir/$template ~/.zshrc"
```
These kinds of usage instructions are generally inappopriate in Nix packages, since the user probably won't see them anyway.

scolobb replied:
If I remove the installation instructions from here, is it a good idea to put them into the meta-attribute `longDescription`?

edolstra commented on pkgs/shells/oh-my-zsh/default.nix:
```diff
@@ -0,0 +1,71 @@
+  '';
+
+  meta = with stdenv.lib; {
+  description = "Oh My Zsh is a framework for managing your zsh configuration.";
```
No periods at the end of descriptions, and don't repeat the package name (i.e. "A framework for managing your zsh configuration" is enough).

scolobb replied:
OK, will fix that shortly.

scolobb commented:
> These kinds of usage instructions are generally inappopriate in Nix packages, since the user probably won't see them anyway. 

I removed installation instructions from `installPhase` of `oh-my-zsh-git` and moved it into the `longDescription` of the package.

> No periods at the end of descriptions, and don't repeat the package name (i.e. "A framework for managing your zsh configuration" is enough).

Fixed.

joachifm commented on pkgs/data/fonts/montserrat/default.nix:
```diff
@@ -0,0 +1,28 @@
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A geometric sans serif font with extended latin support (Regular, Alternates, Subrayada).";
```
Please remove punctuation from the description

scolobb replied:
Oh, sure, forgot to fix that one, thank you.

joachifm commented:
Apart from my nitpick, this looks ready to merge?

scolobb commented:
Removed the trailing full-stop from the description of `montserrat` and rebased on latest `master`.

joachifm commented:
Thank you
